source/dotnet/Test/AdaptiveCards.Templating.Test/AdaptiveCards.Templating.Test.csproj

```diff
@@ -8,7 +8,7 @@
 
   <ItemGroup>
     <PackageReference Include="AdaptiveExpressions" Version="4.11.*" />
-    <PackageReference Include="Antlr4.Runtime.Standard" Version="4.9.1" />
+    <PackageReference Include="Antlr4.Runtime.Standard" Version="4.8.0" />
```

Comment:
The templating library uses version 4.8.0, so the higher version was causing conflicts.

I can also upgrade the templating library to 4.9.1, but I wasn't sure if this would cause errors